### PR TITLE
Fix for playbooks that uses deprecated sub-playbooks

### PR DESCRIPTION
**Pull request by karinafishman**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-9110
realtes: https://jira-dc.paloaltonetworks.com/browse/CIAC-8740

## Description
Updating all the playbooks that use deprecated version of playbooks.
Detonate File - Generic
Ransomware Enrich and Contain
Detonate URL - Generic v1.5
Cortex XDR - Malware Investigation

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by ssokolovich — COMMENTED** [review]
1. General note for RNs - pls mention the new PB name.
2. Pls add the updated PB images to the PR (as you changed some of the tasks' names).
3.  Just make sure that the outputs of the new playbooks are the same as the old ones so it won't brake up the PB logic + they will be set properly also as the parent PB output.

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_File_-_Generic.yml
[the remark this answers is not in the document]
Comment (karinafishman) [reply Packs/CommonPlaybooks/Playbooks/playbook-Detonate_File_-_Generic.yml:1]: We have this conditation in sub-playbooks

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_File_-_Generic.yml [hunk 1]
```diff
@@ -816,202 +823,202 @@ inputs:
   playbookInputQuery:
 outputs:
```
[the remark this answers is not in the document]
Comment (karinafishman) [reply Packs/CommonPlaybooks/Playbooks/playbook-Detonate_File_-_Generic.yml:828]: they are the same

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 2]
```diff
@@ -954,6 +864,53 @@ tasks:
...
+    scriptarguments:
+      Full Report:
+        simple: "True"
+      Interval:
+        simple: "1"
+      Timeout:
+        simple: "10"
+      URL:
+        complex:
+          root: inputs.URL
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:892]: pls add unique transformer here

File: Packs/CortexXDR/ReleaseNotes/6_0_8.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Cortex XDR - Malware Investigation
+
+- The sub-playbook Cortex XDR - Retrieve File v2 was replaced with new version.
```
Comment (ssokolovich) [inline Packs/CortexXDR/ReleaseNotes/6_0_8.md:6]: Add here the replaced sub PB name + the new sub pb name

File: Packs/CommonPlaybooks/ReleaseNotes/2_4_39.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+
+#### Playbooks
+
+##### Detonate File - Generic
+
+- The sub-playbook Detonate File - CrowdStrike Falcon Intelligence Sandbox was replaced with a new version Detonate File - CrowdStrike Falcon Intelligence Sandbox v2.
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/ReleaseNotes/2_4_39.md:6]:
```suggestion
- The sub-playbook "Detonate File - CrowdStrike Falcon Intelligence Sandbox" was replaced with a new version "Detonate File - CrowdStrike Falcon Intelligence Sandbox v2".
```

File: Packs/CommonPlaybooks/ReleaseNotes/2_4_39.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+
+##### Detonate URL - Generic v1.5
+
+- The sub-playbook Detonate URL - CrowdStrike Falcon Intelligence Sandbox was replaced with a new version Detonate URL - CrowdStrike Falcon Intelligence Sandbox v2.
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/ReleaseNotes/2_4_39.md:10]:
```suggestion
- The sub-playbook "Detonate URL - CrowdStrike Falcon Intelligence Sandbox" was replaced with a new version "Detonate URL - CrowdStrike Falcon Intelligence Sandbox v2".
```

**Review by ssokolovich — APPROVED** [review]

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 6]
```diff
@@ -238,51 +238,36 @@ tasks:
...
-      id: 2c83e44b-3ff6-4494-8c99-45870d2b5630
+      id: f2828581-9807-4f24-883e-130c0ca7fec6
       version: -1
-      name: Detonate URL - SecneurX Analysis
-      description: Detonates a URL using the SecneurX Analysis integration. Returns relevant reports to the War Room and file reputations to the context data.
-      playbookName: Detonate URL - SecneurX Analysis
-      type: playbook
-      iscommand: false
+      name: Detonate URL - JoeSecurity
+      description: Submit an URL for sandbox analysis.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:249]:
```suggestion
      description: Submit a URL for sandbox analysis.
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 7]
```diff
@@ -342,34 +325,43 @@ tasks:
...
     task:
       id: 36274297-5bc4-4f17-8f71-eace307e6ffd
       version: -1
-      name: Detonate URL - OPSWAT
-      description: Scan URL resource.
-      script: '|||opswat-filescan-scan-url'
-      type: regular
-      iscommand: true
+      name: Detonate URL - Cuckoo
+      description: Detonating URL with Cuckoo
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:336]:
```suggestion
      description: Detonate a URL with Cuckoo.
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 8]
```diff
@@ -380,105 +372,65 @@ tasks:
...
     task:
       id: 48e0c334-003e-4e26-80df-808e542ad988
       version: -1
-      name: Check if URL exists
-      description: Checks if there is a URL to detonate.
-      type: condition
+      name: Detonate URL - ANYRUN
+      description: |-
+        Detonates one or more URLs using the ANYRUN sandbox integration.
+        Returns relevant reports to the War Room and url reputations to the context data.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:385]:
```suggestion
        Returns relevant reports to the War Room and URL reputations to the context data.
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 9]
```diff
@@ -699,32 +649,32 @@ tasks:
...
     taskid: aaaae5ac-95e9-4542-81c2-a3f634f13437
     type: playbook
     task:
       id: aaaae5ac-95e9-4542-81c2-a3f634f13437
       version: -1
-      name: Detonate URL - Group-IB TDS Polygon
-      description: Detonate URL using Group-IB THF Polygon integration.
-      playbookName: Detonate URL - Group-IB TDS Polygon
+      name: Detonate URL - FireEye AX
+      description: Detonating URL with FireEye AX.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:660]:
```suggestion
      description: Detonates a URL with FireEye AX.
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 10]
```diff
@@ -746,80 +696,43 @@ tasks:
...
-    task:
-      id: e103cfd4-741c-4807-85af-f064c2b4de06
-      version: -1
-      name: Detonate URL - WildFire
-      description: Uploads a URL of a webpage to WildFire for analysis.
-      script: '|||wildfire-upload-url'
-      type: regular
-      iscommand: true
+      name: Detonate URL - Hatching Triage
+      description: Detonating URL with Hatching Triage.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:707]:
```suggestion
      description: Detonates a URL with Hatching Triage.
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 11]
```diff
@@ -954,6 +864,55 @@ tasks:
...
     isautoswitchedtoquietmode: false
+  "21":
+    id: "21"
+    taskid: 60bd5acb-bfa4-411c-89bb-c07a71574ebf
+    type: playbook
+    task:
+      id: 60bd5acb-bfa4-411c-89bb-c07a71574ebf
+      version: -1
+      name: Detonate URL - CrowdStrike Falcon Intelligence Sandbox v2
+      description: Detonate one or more URL's using the CrowdStrike Falcon Intelligence Sandbox integration. This playbook returns relevant reports to the War Room and file reputations to the context data.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:875]:
```suggestion
      description: Detonate one or more URLs using the CrowdStrike Falcon Intelligence Sandbox integration. This playbook returns relevant reports to the War Room and file reputations to the context data.
```

File: Packs/CommonPlaybooks/ReleaseNotes/2_4_39.md [hunk 12]
```diff
@@ -0,0 +1,10 @@
+
+#### Playbooks
+
+##### Detonate File - Generic
+
+- The sub-playbook "Detonate File - CrowdStrike Falcon Intelligence Sandbox" was replaced with a new version "Detonate File - CrowdStrike Falcon Intelligence Sandbox v2".
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/ReleaseNotes/2_4_39.md:6]:
```suggestion
The sub-playbook "Detonate File - CrowdStrike Falcon Intelligence Sandbox" was replaced with a new version "Detonate File - CrowdStrike Falcon Intelligence Sandbox v2".
```

File: Packs/CommonPlaybooks/ReleaseNotes/2_4_39.md [hunk 13]
```diff
@@ -0,0 +1,10 @@
...
+
+##### Detonate URL - Generic v1.5
+
+- The sub-playbook "Detonate URL - CrowdStrike Falcon Intelligence Sandbox" was replaced with a new version "Detonate URL - CrowdStrike Falcon Intelligence Sandbox v2".
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/ReleaseNotes/2_4_39.md:10]:
```suggestion
The sub-playbook "Detonate URL - CrowdStrike Falcon Intelligence Sandbox" was replaced with a new version "Detonate URL - CrowdStrike Falcon Intelligence Sandbox v2".
```

File: Packs/Core/Playbooks/playbook-Ransomware_Enrich_and_Contain.yml [hunk 14]
```diff
@@ -167,8 +171,7 @@ tasks:
       id: 25f7c2e8-f0e2-4ac5-81e8-78eb6cacc629
       version: -1
       name: Robust search of IoCs
-      description: |-
-        Searches Cortex XSIAM indicators. 
+      description: 'Searches Cortex XSIAM indicators. '
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Ransomware_Enrich_and_Contain.yml:174]:
```suggestion
      description: Searches Cortex XSIAM indicators.
```

File: Packs/Core/Playbooks/playbook-Ransomware_Enrich_and_Contain_README.md [hunk 15]
```diff
@@ -1,55 +1,63 @@
 This playbook is responsible for ransomware alert data enrichment and response.
 The playbook executes the following:
 
-1. Checks if the initiator is a remote attacker and allows isolating the remote host, if possible.
+1.Checks if the initiator is a remote attacker and allows isolating the remote host, if possible.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Ransomware_Enrich_and_Contain_README.md:4]:
```suggestion
1. Checks if the initiator is a remote attacker and allows isolating the remote host, if possible.
```

File: Packs/Core/Playbooks/playbook-Ransomware_Enrich_and_Contain_README.md [hunk 16]
```diff
@@ -1,55 +1,63 @@
...
 
-2. Retrieves the WildFire sandbox report and extracts the indicators within it.
+2.Retrieves the WildFire sandbox report and extract the indicators within it.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Ransomware_Enrich_and_Contain_README.md:6]:
```suggestion
2. Retrieves the WildFire sandbox report and extracts the indicators within it.
```

File: Packs/Core/Playbooks/playbook-Ransomware_Enrich_and_Contain_README.md [hunk 17]
```diff
@@ -1,55 +1,63 @@
...
+    * The playbook tries to retrieve the report, but if there is no report available, the playbook tries to fetch the ransomware file for detonation.
 
-   The playbook tries to retrieve the report, but if there is no report available, the playbook tries to fetch the ransomware file for detonation.
-
-3. Hunts for the ransomware alert indicators from the alert table, searches for endpoints that have been seen with them, and allows containing the identified endpoints.
+3.Hunts for the ransomware alert indicators from the alert table, searches for endpoints that have been seen with them, and allows containing the identified endpoints.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Ransomware_Enrich_and_Contain_README.md:9]:
```suggestion
3. Hunts for the ransomware alert indicators from the alert table, searches for endpoints that have been seen with them, and allows containing the identified endpoints.
```

File: Packs/Core/Playbooks/playbook-Ransomware_Enrich_and_Contain_README.md [hunk 18]
```diff
@@ -1,55 +1,63 @@
...
+
 ---
 
 | **Name** | **Description** | **Default Value** | **Required** |
 | --- | --- | --- | --- |
 | isolateRemoteAttacker | Whether to isolate the remote attacker host. | true | Optional |
 | isolateSimilarEndpoints | Whether to isolate endpoints which has been detected with the alert IoCs. | false | Optional |
 | FileSHA256 | The ransomware file SHA256. | alert.initiatorsha256 | Optional |
-| detonateRansomFile | Whether to detonate the ransomware file in sandbox. Set to True to enable file detonation and False to disable it.<br/>By default is set to True | True | Optional |
+| detonateRansomFile | Whether to detonate the ransomware file in sandbox, Set to True to enable file detonation and False to disable it.<br/>By default is set to True | True | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Ransomware_Enrich_and_Contain_README.md:52]:
```suggestion
| detonateRansomFile | Whether to detonate the ransomware file in sandbox. Set to True to enable file detonation and False to disable it.<br/> | True | Optional |
```

File: Packs/Core/ReleaseNotes/3_0_3.md [hunk 19]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Ransomware Enrich and Contain
+
+- Changed the sub-playbook Detonate File - WildFire to a new version.
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/3_0_3.md:6]:
```suggestion
Changed the sub-playbook Detonate File - WildFire to a new version.
```

File: Packs/CortexXDR/ReleaseNotes/6_0_8.md [hunk 20]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Cortex XDR - Malware Investigation
+
+- The sub-playbook "Cortex XDR - Retrieve File" was replaced with a new version "Cortex XDR - Retrieve File v2".
```
Comment (ShirleyDenkberg) [inline Packs/CortexXDR/ReleaseNotes/6_0_8.md:6]:
```suggestion
The sub-playbook "Cortex XDR - Retrieve File" was replaced with a new version "Cortex XDR - Retrieve File v2".
```

**Pull request merged**